mathesar/views/api.py

```diff
@@ -292,3 +301,50 @@ def create(self, request):
         serializer = DataFileSerializer(data=request.data, context={'request': request})
         serializer.is_valid(raise_exception=True)
         return create_datafile(request, serializer.validated_data)
+
+
+class ConstraintViewSet(viewsets.GenericViewSet, ListModelMixin, RetrieveModelMixin, CreateModelMixin, DestroyModelMixin):
```

Comment:
Do we need `CreateModelMixin` and `DestroyModelMixin` if we overwrite those methods?

Reply:
No, we don't. :)